testsuites/syncgateway/functional/tests/test_xattrs.py

```diff
@@ -512,7 +509,6 @@ def test_offline_processing_of_external_updates(params_from_base_test_setup, sg_
     sg_client.verify_docs_in_changes(url=sg_url, db=sg_db, expected_docs=docs_to_verify_in_changes, auth=seth_auth)
```

Comment:
verificy docs in changes is good to be sanity. Is any sanity test covering changes?

Reply:
Marked this test case as sanity